.gitattributes.yml

```diff
@@ -0,0 +1,9 @@
+/benchmark export-ignore
+/bin export-ignore
+/vendor export-ignore
```

Comment:
gitignore is for files that should not go to commits, this is for release control. The purpose is different. Even if tomorrow you decide to add this folder to git, you might still want to exclude from your release.